src/main/java/org/opentripplanner/graph_builder/module/geometry/GeometryAndBlockProcessor.java

```diff
@@ -646,7 +659,12 @@ private LineString getLineStringForShapeId(FeedScopedId shapeId) {
         CoordinateSequence sequence = new PackedCoordinateSequence.Double(coordinates, 2);
         geometry = geometryFactory.createLineString(sequence);
         geometriesByShapeId.put(shapeId, geometry);
-        distancesByShapeId.put(shapeId, distances);
+
+        // If we don't have distances here, we can't calculate them ourselves because we can't
+        // assume the units will match
+        if(hasAllDistances) {
```

Comment:
Should we move this above [here](https://github.com/opentripplanner/OpenTripPlanner/pull/3773/files#diff-865505a36047a98077567c213b125166ad636c1146bf0e5f5b67dfa9cccdcc18R656)